src/main/kotlin/com/kafkakotlin/demo/users/UserController.kt

```diff
@@ -0,0 +1,35 @@
+package com.kafkakotlin.demo.users
+
+import mu.KLogging
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/user")
+class UserController(
+    val userService: UserService
+) {
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    fun createUser(@RequestBody user: User) {
+        return userService.createUser(user)
+    }
+
+    @GetMapping("/all")
```

Comment:
Generally REST convention is to just have the "get all" as `GET /users` or `GET /user`

Reply:
My POST request is /user at the moment. If I changed the GET to /users then I think this is too close to the POST naming and could be confusing. What do you think the POST request should be changed to?

Reply:
It's rest convention that they are the same e.g.

GET /users: returns all users
POST /users: creates a user and returns it (if possible)
PUT /users: would update some or all users dependent on payload
DELETE /users: would delete some or all users dependent on payload

GET /users/123: returns user 123
POST /users/123: only really used if you want the API client to be able to decide what the ID is when creating
PUT /users/123: updates user 123
DELETE /users/123: deletes user 123

Have a read over https://restfulapi.net/resource-naming